documentation/source/user_section/tutorials/spinalcord_segmentation.rst

```diff
@@ -0,0 +1,243 @@
+#. The heatmap is fed into the OptiC algorithm to detect the spinal cord centerline.
+#. The spinal cord centerline is used to extract a patch from the image.
+
+   - We extract a patch to help combat class imbalance. If the full image were to be used instead, the spinal cord region would be small in proportion to the non-spinal cord regions of the image, and thus harder to detect.
+
+#. Lastly, a second convolutional neural network is applied to the extracted patch to segment the spinal cord.
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/spinalcord_segmentation/sct_deepseg_sc_steps.png
+   :align: center
+   :figwidth: 65%
+
+   The steps for ``sct_deepseg_sc``
+
+Example: T1w image
+------------------
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/spinalcord_segmentation/t1_deepseg_before_after.png
+   :align: right
+   :figwidth: 20%
+
+   No leakage with ``sct_deepseg_sc``
+
+Running the script
+^^^^^^^^^^^^^^^^^^
+
+Since we aim to improve the T1 segmentation, ensure that you are still in the T1 directory (``sct_course_london20/single_subject/data/t1``). Once there, run this command:
+
+.. code:: sh
+
+   $ sct_deepseg_sc -i t1.nii.gz -c t1 -qc ~/qc_singleSubj -ofolder deepseg
+
+Much like ``sct_propseg``, we use the same values for ``-i``, ``-c``, and ``-qc``. In this case, however, we have added an additional ``-ofolder`` command. This is so that we do not overwrite the results generated in the previous steps, which allows us to compare the output of both algorithms. ``-ofolder`` is not strictly necessary, however.
+
+Inspecting the results using QC
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once again, you may either execute the command given by the script, or simply refresh the QC webpage from the previous examples.
+
+In this case, ``sct_deepseg_sc`` has managed to improve upon the results of ``sct_propseg``.
+
+Choosing between ``sct_propseg`` and ``sct_deepseg_sc``
+*******************************************************
+
+Although ``sct_deepseg_sc`` was introduced as a follow-up to the original ``sct_propseg``, choosing between the two is not as straightfoward as it may seem. Neither algorithm is strictly superior in all cases; whether one works better than the other is data-dependent. Given the variation in imaging data (imaging centers, sizes, ages, coil strengths, contrasts, scanner vendors, etc.) SCT recommends to try both algorithms with your pilot scans to evaluate the merit of each on your specific dataset, then stick with a single method throughout your study.
+
+Development of these approaches is an iterative process, and the data used to develop these approaches evolves over time. If you have input regarding what has worked (or hasn't worked) for you, we would be happy to hear your thoughts in the `Spinal Cord MRI forums <https://forum.spinalcordmri.org/>`_, as it could help to improve the toolbox for future users.
```

Comment:
```suggestion
Development of these approaches is an iterative process, and the data used to develop these approaches evolves over time. If you have input regarding what has worked (or hasn't worked) for you, we would be happy to hear your thoughts in the `SCT forum<http://forum.spinalcordmri.org/c/sct>`_, as it could help to improve the toolbox for future users.
```